### ISSUE TITLE
Not activating conversation during Freedom factory clearing quest has no effect on its completion

### ISSUE THREAD

**Issue by SurDno** [Decane/SRP#22]
Quest gets completed as soon as every merc on the factory is killed. Though some of them are located just near tunnel and player usually activates conversation while trying to reach them, it really has no effect on its completion, though I previously stated otherwise in #6.

Chekhov just states that the tunnel is already blown up and there's nothing to worry about when it's really not. You can even activate the conversation days after the quest is completed.

So, basically, before it gets completed, you have to do two things:

- Get to the tunnel and activate the radio conversation with tunnel blowing.
- Kill every single merc on the factory.

That would make it more logical.

**Comment by SurDno** [comment]
Task bug?
